src/atomization/atomizer.py

```diff
@@ -278,7 +289,12 @@ def jsonify_content(typ: str) -> list:
     return {typ: jsonify_content(typ) for typ in ["spec", "code", "proof", "spec+code"]}
 
 
-def main():
+# New function to encapsulate CLI logic
+
+
+def run_atomizer(args: list[str] | None = None) -> int:
```

Comment:
Can I see this function broken up / calling smaller functions? isolate DB effects and CLI IO effects, leaving readable business logic.